pages/blog/blog-post.js

```diff
@@ -0,0 +1,301 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {
+  TwitterShareButton,
+  FacebookShareButton,
+} from 'react-share'
+import FaFacebook from 'react-icons/lib/fa/facebook'
+import FaTwitter from 'react-icons/lib/fa/twitter'
+
+import withLayout from '../../components/withLayout'
+import Head from '../../components/Head'
+import { getLocaleFromQuery } from '../../utils/locale'
+import { fetchBlogPost, fetchBlogPostPage } from '../../api/blog'
+import PageSection from '../../components/PageSection'
+import TeamMember from '../../components/TeamMember'
+import teamMemberShape from '../../propTypes/teamMember'
+import imageShape from '../../propTypes/image'
+import Hero from '../../components/Hero'
+import Markdown from '../../components/Markdown'
+import { smBreakpoint, mdBreakpoint, lgBreakpoint } from '../../styling/breakpoints'
+import theme from '../../styling/theme'
+import { rem } from '../../styling/typography'
+import Button from '../../components/Button'
+import Link from '../../components/Link'
+import { RouteNames as routes } from '../../routes'
+
+const BlogPostError = props => (
+  <div>
+    <Head title="FIXME: Oops" description="FIXME: Something went wrong" />
+    <PageSection>
+      <h1>FIXME: Oops</h1>
+      <p className="offset-lg-2 col-lg-8">{props.error}</p>
+    </PageSection>
+  </div>
+)
+
+BlogPostError.propTypes = {
+  error: PropTypes.string,
+}
+
+BlogPostError.defaultProps = {
+  error: null,
+}
+
+const BlogTitle = styled.h2`
+  margin-left: 0;
+  margin-right: 0;
+  text-align: left;
+  width: 100%;
+  color: ${theme.black};
+`
+
+const BlogMarkdown = styled(Markdown)`
+  p:first-child {
+    font-size: ${rem('24px')}
+  }
+
+  p img {
+    width: 100%;
+  }
+
+  @media(min-width:${smBreakpoint}) {
+    p img {
+      margin: 0 -25px;
+      width: calc(100% + 50px);
+    }
+  }
+
+  @media(min-width:${mdBreakpoint}) {
+    p img {
+      margin: 0 -30px;
+      width: calc(100% + 60px);
+    }
+  }
+
+  @media(min-width:${lgBreakpoint}) {
+    p img {
+      margin: 0 -120px;
+      width: calc(100% + 240px);
+    }
+  }
+`
+
+const TeamMemberAuthor = styled(TeamMember)`
+  justify-content: flex-start;
+`
+
+const DateContainer = styled.div`
+  margin-bottom: 1.5rem;
```

Comment:
Use `smallSpacing` from `sizes.js`

Reply:
This goes for the entire PR in general. Try making use of the constants in `sizes.js` wherever you can.